content/efcore/getting-started/loading-related-data.md

```diff
@@ -0,0 +1,153 @@
+# Loading Related Data  
+ 
+> **Note** {.note} 
+> If you want to see the contents of the database or how our entities relate to each other, please refer to the [Database Reference](database-reference.md) and [Model Reference](model-reference) pages. 
+ 
+## Single Layer Inclusion 
+ 
+When loading an entity from the database, EF Core will automatically include properties that are primitive types, but non-primitive types (such as other entities) won't be included. For example, if you run the following code snippet, you will see that the Author field is null: 
```

Comment:
So, author field is null, so if they run the code snippet, instead of "showing that it is null", it throws an error because we try and print `book.Author.FirstName` and `book.Author.LastName`. We need to either change the wording in the tutorial, or change how we're printing right now.